circle.yml

```diff
@@ -42,3 +59,34 @@ workflows:
     jobs:
       - build
       - test
+  linux-build:
+    jobs:
+      # checks out code and installs dependencies once
+      - cypress/install:
+          name: "Linux install"
+          # and builds the app using this command
+          build: "npm run build"
+          post-steps:
+            # show Cypress cache folder and binary versions
+            # to check if we are caching previous binary versions
+            - run: npx cypress cache path
+            - run: npx cypress cache list
+            - run: npx cypress info
+
+      # runs on 1 machine
+      # and records on Cypress Dashboard
+      - cypress/run:
+          executor: pg-cypress
+          name: "1 machine"
+          # job will use workspace with code and dependencies
+          # installed by the "install" job
+          requires:
+            - Linux install
+          record: false
+          parallel: false # load balance all tests
+          parallelism: 1 # across 3 machines
+          group: "1x-electron on CircleCI" # name this group
+          post-install:
+            - run: npm run dbmigratetest up
+          start: "npm run dev" # start server before running tests
```

Comment:
Maybe an `npm run start` would have been a better state to test against?

Reply:
We would need to get the database working with SSL on circleCI before we can make that change (previous attempts didn't go so well)